INSTALL.rst

```diff
@@ -1,19 +1,20 @@
-==================
-Installation Guide
-==================
+############
+Installation
+############
 
-.. note::
+.. toctree::
+   :hidden:
 
-    If you wish to contribute to the project, it's recommended you
-    :ref:`install the latest development version<install_from_source>`.
+   installing_source.rst
 
-.. contents::
 
+==============================
 Installing an official release
 ==============================
 
-Matplotlib and its dependencies are available as wheel packages for macOS,
-Windows and Linux distributions::
+Matplotlib releases are available as wheel packages for macOS, Windows and
+Linux on `PyPI <https://pypi.org/project/matplotlib/>`_. Install it using
+``pip``::
```

Comment:
this could also be tagged with `.. code-block:: sh`, or you could just use https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-highlight at the top instead.